api/bootstrap.proto

```diff
@@ -231,6 +232,7 @@ message Bootstrap {
     // to know how to speak to the management server. These cluster definitions
     // may not use EDS (i.e. they should be static IP or DNS-based).
     repeated Cluster clusters = 2;
+    repeated Secret secrets = 3;
```

Comment:
@htuch Thoughts on placement of `repeated` fields?

    message StaticResources {
      repeated Listener listeners = 1;
      repeated Cluster clusters = 2;
      repeated Secret secrets = 3;
    }
    
    message Secret {
      TlsCertificate tls_certificate = 1;
    }

vs

    message StaticResources {
      repeated Listener listeners = 1;
      repeated Cluster clusters = 2;
      Secret secrets = 3;
    }
    
    message Secret {
      repeated TlsCertificate tls_certificates = 1;
    }

vs

    message StaticResources {
      repeated Listener listeners = 1;
      repeated Cluster clusters = 2;
      repeated Secret secrets = 3;
    }
    
    message Secret {
      repeated TlsCertificate tls_certificates = 1;
    }

Note that `Secret` will contain more than just `TlsCertificate` objects in the future.

Reply:
Option 1 is cleaner, since when we have a `DiscoveryResponse`, it already is a repeated list of resources, with the caveat below about the recursive definition.